utils/yoke/src/yoke.rs

```diff
@@ -93,7 +93,9 @@ impl<Y: for<'a> Yokeable<'a>, C: StableDeref> Yoke<Y, C> {
     ///
     /// # Examples
     ///
-    /// ```
+    /// ```ignore
+    /// # // ignored because while it works on stable, our coverage CI is still on an old
```

Comment:
@sffc should we update coverage CI too? I think this is a good chance to do so.

Reply:
I guess we could pin a differently nightly for coverage than for FFI